src/actor/character/sheets-tabs/main-tab.hbs

```diff
@@ -1,10 +1,31 @@
 <div class="main-tab">
   <div class="left-content">
+    <div class="initiative-modifier">
+      <h4>
+        {{localize "BIO.INITIATIVE_MODIFIER"}}
+        <input id="number-input" type="number" name="system.baseAttribute.initiative_modifier.value" value="{{this.actor.attributes.initiative_modifier.value}}" />
```

Comment:
Mind switching number-input to a class? I don't remember why I made it an id, but I'm guessing it was a mistake.

Reply:
Mind if I do that in a followup? I'm now noticing that the other id (`calculated-attribute-value`) should also be a class

Reply:
Addressed here: https://github.com/AllPurposeName/fabula-ultima-foundry-vtt/pull/49